gpii/node_modules/singleInstance/test/SingleInstanceTests.js

```diff
@@ -0,0 +1,207 @@
+/*
+ * singleInstance Tests
+ *
+ * Copyright 2017 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * The R&D leading to these results received funding from the
+ * Department of Education - Grant H421A150005 (GPII-APCP). However,
+ * these results do not necessarily represent the policy of the
+ * Department of Education, and you should not assume endorsement by the
+ * Federal Government.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("universal"),
+    fs = require("fs"),
+    os = require("os"),
+    child_process = require("child_process");
+
+var jqUnit = fluid.require("node-jqunit");
+var gpii = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.tests.singleInstance");
+
+require("singleInstance");
+
+jqUnit.module("gpii.tests.singleInstance");
+
+fluid.defaults("gpii.tests.singleInstance.testData", {
+    gradeNames: "fluid.modelComponent",
+    model: {
+        tests: [
```

Comment:
Since these test definitions themselves aren't mutable, they don't need to go into the model - they can just go into a global constant e.g. gpii.tests.singleInstance.testDefs, with fluid.freezeRecursive applied